### PR TITLE
grb unstable init, vul 2020-02-15 -> 2022-07-02

### PR DESCRIPTION
**Pull request by cafkafk**
###### Description of changes
- init grb unstable at 2022-07-02
- update vul 2020-02-15 -> 2022-07-02
- add self as maintainer for both and kjv

grb, vul, and kjv are essentially the same package, so it makes
sense to handle them in unison.

###### For maintainers
These packages could all easily go in a shared directory as
they are very closely related.

Before that, it would make sense to make kjv reflect upstream instead
of the fork it currently reflects (e.g. create a version
for the fork and for the original) before doing this.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -2744,6 +2746,7 @@ with pkgs;
 
   kjv = callPackage ../applications/misc/kjv { };
 
+
```
Comment (AndersonTorres) [inline pkgs/top-level/all-packages.nix:2749]: ??

Comment (cafkafk) [reply pkgs/top-level/all-packages.nix]: Just a careless mistake 😅

File: pkgs/top-level/all-packages.nix [hunk 2]
```diff
@@ -2726,6 +2726,8 @@ with pkgs;
 
   goku = callPackage ../os-specific/darwin/goku { };
 
+  grb = callPackage ../applications/misc/grb { };
```
Comment (AndersonTorres) [inline pkgs/top-level/all-packages.nix:2729]: Split the commit TODO:

[ ] The new package (including this `grb = callPackage ...` line and the new file) should be in a separate commit.
[ ] Each package you have updated should be in its own separate commit.

Otherwise the code is OK.

**Review by AndersonTorres — CHANGES_REQUESTED** [review]

File: pkgs/top-level/all-packages.nix (hunk 2 above)
Comment (cafkafk) [reply pkgs/top-level/all-packages.nix:2729]: I did this + updated my maintainer-list.nix info in another separate commit

File: maintainers/maintainer-list.nix [hunk 3]
```diff
@@ -15186,13 +15186,18 @@
     }];
   };
   cafkafk = {
-    email = "cafkafk@cafkafk.com";
+    email = "christina@cafkafk.com";
```
Comment (AndersonTorres) [inline maintainers/maintainer-list.nix:15189]: Reword commit:

`maintainer-list.nix: update cafkafk entry`

**Review by AndersonTorres — CHANGES_REQUESTED** [review]
Sorry by the annoyances.

**Comment by cafkafk** [comment]
> Sorry by the annoyances.

Nw, it's okay.

The only thing is that the way I did multiple keys is the same way that cab404, cge, corbanr, cypherpunk2140, felschr, jmc-figueira, primeos, and risson did it in maintainer-list.nix. 

I think it's better to keep it like those other maintainers have written multiple key fingerprints for consistency.

I'll change the commit messages now, if you still think the syntax should be changed I can also do that but just let me know.

**Comment by cafkafk** [comment]
Somehow it ran out of space, I think the package should still work fine thou. 

https://gist.github.com/GrahamcOfBorg/918d84c75d6a8c8c7e1180c21224be1e

**Comment by AndersonTorres** [comment]
@ofborg eval

**Review by AndersonTorres — APPROVED** [review]
LGTM. Remind me tomorrow if ofBorg fails.

**Comment by cafkafk** [comment]
Result of `nixpkgs-review pr 189873` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>grb</li>
    <li>vul</li>
  </ul>
</details>

**Comment by cafkafk** [comment]
https://gist.github.com/GrahamcOfBorg/182f4cb5998dfd7aa718575fb13899d2
> error: creating file '/nix/store/add-3559962-0/x/.git/objects/71/88af340959d730cb96295bb6b77ec8c5110541': No space left on device

Ofborg fails saying it ran out of space. This has been happening with multiple repos since yesterday.

**Comment by cafkafk** [comment]
@ofborg eval

**Comment by AndersonTorres** [comment]
Full passed now.

About your suggestion, maybe it would be useful to create some abstraction around these bible-tools.
